ketcher-autotests/tests/User-Interface/Zoom-Changes/zoom-changes.spec.ts

```diff
@@ -44,4 +59,100 @@ test.describe('Zoom changes', () => {
 
     expect(zoomInputValue).toBe('80%');
   });
+
+  test('Zoom In button verification', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-1761
+    */
+    await pressButton(page, STRUCTURE_LIBRARY_BUTTON_NAME);
+    await page.getByRole('tab', { name: 'Functional Groups' }).click();
+    await selectFunctionalGroups(FunctionalGroups.CO2Et, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await resetCurrentTool(page);
+
+    await takeTopToolbarScreenshot(page);
```

Comment:
If you want to check the value in zoom input, then I propose to use the approach above – you can directly get the value from input and you don't need to take a screenshot.